pkg/cmd/util/tokencmd/request_token.go

```diff
@@ -30,7 +30,7 @@ const (
 	// See IETF Draft:
 	// https://tools.ietf.org/html/draft-ietf-oauth-discovery-04#section-2
 	// Copied from pkg/cmd/server/origin/nonapiserver.go
-	oauthMetadataEndpoint = "/.well-known/oauth-authorization-server"
+	OauthMetadataEndpoint = "/.well-known/oauth-authorization-server"
```

Comment:
Do not export this, it is already a copy from a different file.  Hard code the string, it can never change.